osmaxx-py/osmaxx/excerptexport/services/conversion_api_client.py

```diff
@@ -171,13 +180,41 @@ def update_order_status(self, extraction_order):
             if progress == 'successful':
                 self._download_result_files(extraction_order, job_status)
 
+    def get_country_list(self):
+        def fetch_countries():
+            self.login()
+            response = self.authorized_get(self.country_base_url)
+            if self.errors:
+                logger.error('could not fetch country list: ', self.errors)
+                return self.errors
```

Comment:
Wouldn't it be saner to raise an exception, here? Or at least to return an empty collection? Unaware callers of this function (including all its callers introduced in this pull request) would otherwise treat the errors as countries!